README.md

```diff
@@ -1,37 +1,41 @@
 # FaunaDB
 
-VSCode extension for FaunaDB.
+VS Code extension for FaunaDB.
 
-## Features
+## Getting Started
 
-### Browse your database data
+To get started using the Fauna VS Code extension, you'll need to set your secret key in order to access database information. Keys can be created in the Fauna Console webapp or via the Shell CLI.
 
-You can browse databases, indexes, collections, documents and functions.
+To set the key, go to `Code > Preferences > Settings > Extensions > FaunaDB`.
 
-![Browser your database data](media/browse-feature.png)
+![Extension settings](media/extension-settings.png)
 
-### Run queries
+* `faunadb.secretKey`: Your database secret.
 
-![Run queries](media/query-feature.gif)
+> WARNING: Be careful! To avoid exposing this secret, do **not** commit your local `.vscode` configuration.
+
+## Features
 
 **Commands**
 
 * FaunaDB: Create query
 * FaunaDB: Run query
 
-## Extension Settings
+### Browse database
 
-Before you start you need to set your secret on Code > Preferences > Settings > Extensions > FaunaDB.
+With this extension, you can browse the Databases, Indexes, Collections, Documents and Functions associated with your Fauna database right inside of the VS Code sidebar.
```

Comment:
"Databases, Indexes, Collections, Documents and Functions" can all be lowercase

Reply:
@n400 Will revise. I capitalized them to match our documentation, but since they're not necessarily proper nouns lowercase should suffice. Thanks!

Reply:
i only see them capitalized when they are in links, maybe because they're referring to titles of articles. do you see them capitalized elsewhere? i'll ask ewan how it works